doc/common/glossary.md

```diff
@@ -56,6 +56,8 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **Virtual Application (VA):** A general term for software which can be loaded into a Virtual Machine.
   >_*Note:*_ a **VNF** is one type of VA.
 
+- **Virtual CPU (vCPU):** Represents a portion of the host's computing resources allocated to a virtualised resource, for example, to a virtual machine or a container. One or more vCPUs can be assigned to a  virtualised resource.
```

Comment:
This definition is too wide. It applies to many other things, including a physical CPU in an SMP system or a Linux process.

Reply:
Any suggestions how to make it more narrow?